<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SHARMAPC-PC" name="RetryMultipleFaliedtestCases" tests="3" failures="2" timestamp="23 Apr 2020 14:21:34 GMT" time="0.04" errors="0">
  <testcase name="Test1" time="0.013" classname="tests.TestRetryLogic">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at tests.TestRetryLogic.Test1(TestRetryLogic.java:11)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase name="Test2" time="0.001" classname="tests.TestRetryLogic">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at tests.TestRetryLogic.Test2(TestRetryLogic.java:16)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
  <testcase name="Test3" time="0.001" classname="tests.TestRetryLogic"/>
</testsuite> <!-- RetryMultipleFaliedtestCases -->
